indico/modules/events/surveys/templates/emails/start_notification_email.txt

```diff
@@ -0,0 +1,14 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    Survey has started
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+    The Survey "{{ survey.title }}" is now open. From now on, answers can be submitted to the survey.
```

Comment:
survey lowercase, "You may now answer it" might be better in the second sentence (and i'd put it in a new line).